static/js/pages/elections.js

```diff
@@ -30,7 +30,7 @@ var independentExpenditureColumns = [
         return {
           support_oppose_indicator: row.support_oppose_indicator,
           candidate_id: row.candidate_id,
-          is_notice: false,
+          is_notice: false
```

Comment:
Oops, sorry. We can leave it because it was there and we'll want to use it in the future, but it won't actually make a difference now will it?